scripts/diff.js

```diff
@@ -0,0 +1,55 @@
+const exec = require('child_process').execSync;
+const spawn = require('child_process').spawnSync;
+const fs = require('fs');
+
+let packages = JSON.parse(exec('yarn workspaces info --json').toString().split('\n').slice(1, -2).join('\n'));
+let uuid = require('uuid')();
+
+let ksdiff = process.argv.includes('--ksdiff');
```

Comment:
use yargs? we already have it in the repo

Reply:
seems like overkill for two flags, no? 😉

Reply:
yeah, but it includes the nice help flag I think listing out the options

Reply:
Follow up if we add more flags.